imports/plugins/included/jobcontrol/server/jobs/exchangerates.js

```diff
@@ -22,13 +22,23 @@ export function setupFetchFlushCurrencyHooks() {
           backoff: "exponential" // delay by twice as long for each subsequent retry
         })
         .repeat({
-          schedule: later.parse.text(refreshPeriod)
+          schedule: Jobs.later.parse.text(refreshPeriod)
         })
         .save({
           // Cancel any jobs of the same type,
           // but only if this job repeats forever.
           cancelRepeats: true
         });
+
+      // Run the first time immediately after server start. The repeat({schedule}) option via
+      // later.js scheduling won't run before the first scheduled time, even with delay() or after()
+      new Job(Jobs, "shop/fetchCurrencyRates", {})
+        .priority("normal")
```

Comment:
If we're going to standardize on the version of `later` included with the job queue package, we should probably remove the one from Reaction's package.json.  Since `@reactioncommerce/job-queue` is now on npm and [already has `later` defined as a dependency](https://github.com/reactioncommerce/reaction-job-queue/blob/master/package.json#L27), we shouldn't need to require `later` again and inject that dependency [here](https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/job-collection/lib/index.js#L6-L8) (right @mikemurray?).  But that's outside the scope of this issue and probably needs an issue to be filed over at https://github.com/reactioncommerce/reaction-job-queue.  Either way, I deleted `later` from Reaction's `package.json` and confirmed everything still works.  It was only used for the jobs and the job queue package installs it for you already, so I think you can safely remove it in this PR.

Otherwise, I can confirm that the fix works, so I think we're good to go.  @mikemurray can you just confirm how that `later` stuff is being handled in the npm package?  Did you do that dependency injection to solve some kind of issue?